packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -20,7 +20,11 @@ import 'view.dart';
 export 'package:flutter/gestures.dart' show HitTestResult;
 
 /// The glue between the render tree and the Flutter engine.
-abstract class RendererBinding extends BindingBase implements SchedulerBinding, ServicesBinding, HitTestable {
+abstract class RendererBinding extends BindingBase with SchedulerBinding, ServicesBinding, HitTestable {
```

Comment:
HitTestable.hitTest() is abstract, which means that reference to super.hitTest() at line 287 below ends up flagged as unresolved by DFE.